(DOCSP-44659) addressing feedback from stakeholders (#102)
mongodb/docs-atlas-architecture: source/network-security.txt

databases.

> We recommend using M10+ dedicated {+clusters+} because all {+service+} projects with one or more M10+ dedicated {+clusters+} receive

Reviewer: ```suggestion
We recommend using M10+ dedicated {+clusters+} because all {+service+} projects with one or more M10+ dedicated {+clusters+} receive their own dedicated:
```

> their own dedicated |vpc| on {+aws+} or {+gcp+}, (or {+vnet+}, if you use |azure|).

Reviewer: ```suggestion
- |vpc| on {+aws+} or {+gcp+}.
- {+vnet+} on |azure|.
```

certain topology changes, such as a :ref:`conversion from replica set
to sharded cluster <scale-cluster-sharding>`, the
:ref:`addition of shards <scale-cluster-shardNum>`, or a :ref:`region
change <scale-cluster-region>` require that you use new IP addresses. 

In the case of converting from a replica set to a sharded cluster, the 

> failure to reconnect the application clients may cause your application 

Reviewer: ```suggestion
failure to reconnect the application clients might cause your application 
```

> to suffer from data outages. If you are using a |dns| seed list 
> connection string, your application automatically connects to the |mongos| 
> for your sharded cluster. If you are using a standard connection string, 
> you must update your connection string to reflect your new cluster topology.

Reviewer: ```suggestion
to suffer from data outages. If you use a |dns| seed list 
connection string, your application automatically connects to the |mongos| 
for your sharded cluster. If you use a standard connection string, 
you must update your connection string to reflect your new cluster topology.
```
